src/metabase/api/card.clj

```diff
@@ -181,7 +181,6 @@
   (let [raw-card (db/select-one Card :id id)
         card (-> raw-card
                  (hydrate :creator
-                          :bookmarked
```

Comment:
Sorry, I forgot to call this out. There is actually no hydration method for `:bookmarked` anymore. So I removed this because it's a no-op in Toucan 1. In Toucan 2, it's an error